digitalforms-ords-client/src/test/java/ca/bc/gov/open/pssg/rsbc/digitalforms/ordsclient/health/HealthServiceImplTest.java

```diff
@@ -24,7 +24,7 @@
 public class HealthServiceImplTest {
 
     public static final String API_EXCEPTION = "api exception";
-    private HealthServiceImpl sut;
```

Comment:
'Sut' was Alex. I think this is French for something. I never did find out? Ha..